packages/api/src/graphql/invoice.ts

```diff
@@ -83,7 +84,11 @@ export const GraphQLPublicInvoice = new GraphQLObjectType<Invoice, Context>({
 export const GraphQLinvoiceFilter = new GraphQLInputObjectType({
   name: 'InvoiceFilter',
   fields: {
-    mail: {type: GraphQLString}
+    mail: {type: GraphQLString},
+    paidAt: {type: GraphQLDate},
+    canceledAt: {type: GraphQLDate},
```

Comment:
`cancelledAt` or `canceledAt` ? :)

Reply:
both of them. one L is more common in American English.

https://www.merriam-webster.com/words-at-play/canceled-or-cancelled#:~:text=While%20both%20canceled%20and%20cancelled,more%20common%20in%20British%20English.